source/graphql.ts

```diff
@@ -0,0 +1,107 @@
+import express from 'express';
+import { ApolloServer, gql } from 'apollo-server-express';
+import { readFixtureSync } from './files';
+
+function useResolverWithFixture(prefix: String) {
+  return function useResolverWithFixtureResolver(
+    _: any,
+    __: any,
+    ___: any,
+    info: any
+  ) {
+    const { fieldName } = info;
+    return readFixtureSync(prefix + fieldName + '.json');
+  };
+}
+
+function createQueryResolver(carrier: {}, field: any) {
+  const {
+    name: { value },
+  } = field;
+  return {
+    ...carrier,
+    [value]: useResolverWithFixture('graphql/queries/'),
+  };
+}
+
+function createMutationResolver(carrier: {}, field: any) {
+  const {
+    name: { value },
+  } = field;
+  return {
+    ...carrier,
+    [value]: useResolverWithFixture('graphql/mutations/'),
+  };
+}
+
+function findQueryDefinitions(definition: any) {
+  return definition.name.value === 'Query';
+}
+
+function findMutationDefinitions(definition: any) {
+  return definition.name.value === 'Mutation';
+}
+
+export class GraphQLManager {
+  private definitions: any;
```

Comment:
Rename to something like `typeDefs`, `gqlSchema`, or `apolloSchema` (avoiding `this.definitions.definitions`).